File: KM.Domain/Entities/VipPackage.cs
namespace KM.Domain.Entities
{
    public class VipPackage : BaseEntity
    {
        public required string Name { get; set; }
        public decimal Price { get; set; }
        public decimal PriceSell { get; set; }
        public int DurationDay { get; set; }
        public required string Description { get; set; }
        public bool IsDelete { get; set; } = false;
    }
}

File: client/src/app/shared/models/vip-package.ts
export interface VipPackage {
  id: number;
  name: string;
  price: number;
  priceSell: number;
  durationDay: number;
  description: string;
}

export interface VipPackageCreate {
  name: string;
  price: number;
  priceSell: number;
  durationDay: number;
  description: string;
}
